### ISSUE TITLE
No longer available on CRAN

### ISSUE THREAD

**Issue by thijsjanzen** [Matherion/userfriendlyscience#19]
It seems the package has been removed from CRAN:

> Package ‘userfriendlyscience’ was removed from the CRAN repository.

> Formerly available versions can be obtained from the archive.

> Archived on 2021-01-24 as requires archived package 'ufs'.

See here : https://cran.r-project.org/web/packages/userfriendlyscience/index.html

**Comment by Lucas-Czarnecki** [comment]
Is there a reason why it was removed? I found it very useful for post-hoc testing!

**Comment by Matherion** [comment]
Yes, I'm sorry. This package has become very unwieldy, which is why its functionality is being moved to different packages, specifically:

- ufs (general-purpose 'replacement', https://r-packages.gitlab.io/ufs)
- rosetta (for teaching statistics; https://r-packages.gitlab.io/rosetta/
- behaviorchange (for behavior change functions; https://r-packages.gitlab.io/behaviorchange)
- limonaid (for working with LimeSurvey; https://r-packages.gitlab.io/limonaid)

I'll try to make time to replace these functions with imports from those packages and push userfriendlyscience again, but in the meantime, 'ufs' might have what you need?

**Comment by Lucas-Czarnecki** [comment]
Thanks @Matherion. Maintaining is a thankless job sometimes. I appreciate the information and am glad to hear that userfriendlyscience lives on in other packages! I have found `rstatix` useful on my end. I will have to check out `ufs` (good name). Thank you!
